doc/source/schema/schema.md

```diff
@@ -36,33 +39,79 @@ The terminology used in this specification is based on [xarray's terminology](ht
 `InferenceData` stores all quantities that are relevant to fulfilling its goals in different groups. Different groups generally distinguish conceptually different quantities in Bayesian inference, however, convenience in {ref}`creation <creating_InferenceData>` and {ref}`usage <working_with_InferenceData>` of `InferenceData` objects also plays a role. In general, each quantity (such as posterior distribution or observed data) will be represented by several multidimensional labeled variables.
 
 ### Rules
-Following are a few rules which should be followed:
+Below are a few rules that should be followed:
 * Each group should have one entry per variable and each variable should be named.
-* When relevant, the first two dimensions of each variable should be the sample identifier (`chain`, `draw`).
-* For groups like `observed_data` or `constant_data`, the two initial dimensions(`chain`, `draw`) are omitted.
+* Dimension names `chain`, `draw`, `sample` and `pred_id` are reserved for
```

Comment:
Oh, you know, `pred_id` is not defined anywhere. IIRC, we discussed using this when one needs multiple IID predictions per sample. Perhaps add a note about this to `posterior_predictive`?

Reply:
will do, thanks for pointing it out